tests/unit/lms/services/course_test.py

```diff
@@ -163,11 +163,23 @@ def test_get_from_launch_when_new_and_historical_course_exists(
 
     @pytest.mark.parametrize(
         "custom_course_starts, course_starts_at",
-        [(None, None), ("2022-01-01T00:00:00Z", datetime(2022, 1, 1, tzinfo=UTC))],
+        [
+            (None, None),
+            ("NOT A DATE", None),
```

Comment:
This was a test case missing from a previous PR.

We need to test that invalid dates are also handled. Coverage doesn't catch it because it's handled in the same line as the `None` case.

Added it here in a separate commit.